client/src/components/EmailFrontend.js

```diff
@@ -50,12 +50,12 @@ const EnumField = ({
   selected_enums_for_field,
   disabled,
   state_update_callback,
-  classes,
 }) => (
-  <div className={`${form_type} ${classes}`}>
+  <div className={`${form_type}`}>
```

Comment:
Unnecessary template literal

Reply:
Unless `form_type` can possibly be undefined? I don't think the email backend allows that, but could always use the `classnames` util to be safe.